### PR TITLE
Pull in Jesse's large model fix

### PR DESCRIPTION
**Pull request by sbhaktha**
@jkinkead : Since we cannot indefinitely wait for the authors to merge your PR, I've created an `allenai` fork of Word2VecJava. It's set up to build/release with SBT. This PR merges your fixes to support large models (from https://github.com/medallia/Word2VecJava/pull/29/files).

Once this is merged I will release this to Maven so that IKE's dependency can be changed to the public version.

### CODE REVIEW

**Comment by jkinkead** [comment]
I'll just call this good - I'm assuming you ran the tests?

LGTM!

**Comment by sbhaktha** [comment]
@jkinkead : Actually I realized I didn't have the right library included to run tests in SBT. Changed `build.sbt`. Could you look at this morning's commit? All tests passed:

```
...
Stage TRAIN_NEURAL_NETWORK, progress 0.7580901473562555%
[info] Passed: Total 13, Failed 0, Errors 0, Passed 13
[success] Total time: 12 s, completed May 9, 2016 10:04:40 AM
```

File: build.sbt [hunk 1]
```diff
@@ -7,10 +7,11 @@ import sbtrelease.ReleaseStateTransformations._
 name := "Word2VecJava"
 
 libraryDependencies ++= Seq(
+  "org.scalacheck" %% "scalacheck" % "1.12.0" % Test,
```
Comment (jkinkead) [inline build.sbt]: I usually put `test` deps at the end, but LGTM otherwise! :)

**Comment by sbhaktha** [comment]
ok switched em. :) Merging.

**Pull request merged**